app/scripts/modules/core/delivery/status/executionStatus.component.ts

```diff
@@ -8,8 +8,8 @@ import './executionStatus.less';
 
 export class ExecutionStatusController implements IComponentController {
   public execution: IExecution;
-  public toggleDetails: () => void;
-  public showingDetails: () => boolean;
+  public toggleDetails: (node: any) => void;
```

Comment:
super super cool if this can be an interface (either inlined or explicitly declared/exported somewhere)